Eatery Blue/UI/HomeScreen/HomeModelController.swift

```diff
@@ -71,25 +71,27 @@ class HomeModelController: HomeViewController {
             updateCellsFromState()
         }.store(in: &cancellables)
     }
-
-    private func updateAllEateriesFromNetworking() async {
-        do {
-            let eateries = isTesting ? DummyData.eateries : try await Networking.default.eateries.fetch(maxStaleness: 0)
-            allEateries = eateries.filter { eatery in
-                return !eatery.name.isEmpty
-            }.sorted(by: {
-                if $0.isOpen == $1.isOpen {
-                    $0.name < $1.name
-                } else {
-                    $0.isOpen
+    
+    private func updateSimpleEateriesFromNetworking() async {
+            do {
+                let eateries = isTesting ? DummyData.eateries : try await Networking.simple.eateries.fetch(maxStaleness: 0)
+                if isLoading{
+                    allEateries = eateries.filter { eatery in
+                        return !eatery.name.isEmpty
+                    }.sorted(by: {
+                        if $0.isOpen == $1.isOpen {
+                            return $0.name < $1.name
+                        }
+                        return $0.isOpen
```

Comment:
Lowkey you could do `$0.isOpen == $1.isOpen ? ($0.name < $1.name) : $0.isOpen`
Not sure if that will work though